vendor/bat-native-ledger/src/bat/ledger/internal/contribution/contribution_tip.cc

```diff
@@ -74,6 +75,7 @@ void ContributionTip::ServerPublisher(
   queue_list.push_back(std::move(publisher));
 
   auto queue = ledger::ContributionQueue::New();
+  queue->id = base::GenerateGUID();
```

Comment:
normally queue id is generated right before it's inserted. Problem is that one time tip skip normal queue flow as we don't want to delay them like AC or monthly tips. Because of that we need to generate id when object is created so that queue can be than deleted correctly